<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>806</width>
    <height>683</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>假命令行</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>新宋体</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>制作：yukigamau|版本：1.1</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QGroupBox" name="pageSelectBox">
        <property name="title">
         <string/>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QRadioButton" name="schoolPageButton">
           <property name="text">
            <string>校园服务(a)</string>
           </property>
           <property name="shortcut">
            <string>A</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="sqlitePageButton">
           <property name="text">
            <string>sqlite3(b)</string>
           </property>
           <property name="shortcut">
            <string>B</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="vscodePageButton">
           <property name="text">
            <string>VSCode(c)</string>
           </property>
           <property name="shortcut">
            <string>C</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="mainShowWidget">
      <widget class="QWidget" name="schoolPage">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="2" column="3">
           <widget class="QLineEdit" name="chooseClassText"/>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>办事大厅：</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="websiteButton">
            <property name="text">
             <string>打开上网登录页面(d)</string>
            </property>
            <property name="shortcut">
             <string>D</string>
            </property>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="QLineEdit" name="hallText"/>
          </item>
          <item row="3" column="0">
           <widget class="QPushButton" name="hallButton">
            <property name="text">
             <string>打开办事大厅(f)</string>
            </property>
            <property name="shortcut">
             <string>F</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="websiteLabel">
            <property name="text">
             <string>上网登录网址：</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QCheckBox" name="shutdownBox">
            <property name="text">
             <string>操作后关闭程序(h)</string>
            </property>
            <property name="iconSize">
             <size>
              <width>12</width>
              <height>12</height>
             </size>
            </property>
            <property name="shortcut">
             <string>H</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QLineEdit" name="websiteText"/>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="chooseClassButton">
            <property name="text">
             <string>打开选课网站(e)</string>
            </property>
            <property name="shortcut">
             <string>E</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="chooseClassLable">
            <property name="text">
             <string>选课页面：</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QPushButton" name="documentButton">
            <property name="text">
             <string>打开公文通(g)</string>
            </property>
            <property name="shortcut">
             <string>G</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>公文通地址：</string>
            </property>
           </widget>
          </item>
          <item row="4" column="3">
           <widget class="QLineEdit" name="documentText"/>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="sqlitePage">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QGridLayout" name="gridLayout_7">
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="0" column="2">
           <widget class="QComboBox" name="dbFilesChangeCCB"/>
          </item>
          <item row="4" column="0">
           <widget class="QPushButton" name="runSqlButton">
            <property name="text">
             <string>执行(ctrl+r)</string>
            </property>
            <property name="shortcut">
             <string>Ctrl+R</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>表格名称：</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_8">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>数据库地址：</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="openSqlViewButton">
            <property name="text">
             <string>表视图(F5)</string>
            </property>
            <property name="shortcut">
             <string>F5</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>输入代码↓</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLineEdit" name="tableNameText"/>
          </item>
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QPushButton" name="dbFilesChangeButton">
              <property name="text">
               <string>修改数据库名单(d)</string>
              </property>
              <property name="shortcut">
               <string>D</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="sortCCBButton">
              <property name="text">
               <string>排序(e)</string>
              </property>
              <property name="shortcut">
               <string>E</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="runSqlText">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>表视图↓</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="saveButton">
            <property name="text">
             <string>应用修改(ctrl+s)</string>
            </property>
            <property name="shortcut">
             <string>Ctrl+S</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="dbTable"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="vscodePage">
       <layout class="QGridLayout" name="gridLayout_8">
        <item row="4" column="0">
         <widget class="QPushButton" name="openVSCodeButton">
          <property name="text">
           <string>配置MSVC环境打开VSCode(g)</string>
          </property>
          <property name="shortcut">
           <string>G</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="folderDelButton">
          <property name="text">
           <string>删除文件夹链接(f)</string>
          </property>
          <property name="shortcut">
           <string>F</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="folderAddText"/>
        </item>
        <item row="6" column="0">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="folderAddButton">
          <property name="text">
           <string>添加文件夹链接</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="foldersCCB"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>文件夹：</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QCheckBox" name="shutdownForVSCodeBox">
          <property name="text">
           <string>开启后关闭程序(h)</string>
          </property>
          <property name="shortcut">
           <string>H</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>MSVC命令行地址：</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="vsDevCmdAddrText"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="dbFilesChangePage">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="5">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="0">
           <widget class="QPushButton" name="backButton">
            <property name="text">
             <string>返回(ESC)</string>
            </property>
            <property name="shortcut">
             <string>Esc</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="operateCCB">
            <item>
             <property name="text">
              <string>新建</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>删除</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>改名</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QPushButton" name="operateButton">
            <property name="text">
             <string>执行(r)</string>
            </property>
            <property name="shortcut">
             <string>R</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_10">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>选择操作：</string>
            </property>
           </widget>
          </item>
          <item row="1" column="4">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>修改后不可撤回</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QStackedWidget" name="operatorPage">
          <widget class="QWidget" name="newOperator">
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="1">
             <widget class="QLineEdit" name="newDBNameText"/>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_12">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>新建文件的名称：
要带上后缀名
请使用标准的地址写法</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <spacer name="verticalSpacer_3">
              <property name="orientation">
               <enum>Qt::Orientation::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="deleteOperator">
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="1" column="0">
             <widget class="QListWidget" name="DBList">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>选择要删除的文件：</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="renameOperator">
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="0" column="2">
             <widget class="QComboBox" name="renameCCB"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_15">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>重命名的名字：</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <spacer name="verticalSpacer_5">
              <property name="orientation">
               <enum>Qt::Orientation::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="1" column="2">
             <widget class="QLineEdit" name="renameText"/>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_14">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>选择要重命名的文件：</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QCheckBox" name="addrSaveButton">
              <property name="text">
               <string>在原路径下修改</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>如果取消勾选，请保证输入的名字包含地址信息。
若勾选且在名字中包含路径信息，将仍然修改其路径。</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
